README.md

```diff
@@ -17,7 +17,7 @@ Invite dialog example:
 
 How to compile
 ============
-This plugin works with JIRA 7 (tested with 7.1.7 & 7.2.1).
+This plugin works with Jira 7 (tested with 7.1.7 & 7.2.1).
```

Comment:
For my understanding to validate the PR, did you have a chance to test it on Jira 7.6?

nice to have/not blocking: if so, could be updated here under "tested with"

Reply:
@arnaudj  
Sorry, I forgot to notify you about deployed test instance. 
Also, keep in your mind, between 7.2 and 7.6 does not have method changes which we are using. 

Anyway, I have sent to directly the link ot test instance

Reply:
lgtm codewise, thanks for the PR
testing on UAT with your link now, let's liaise with slack

After testing is confirmed ok, I'll let @fsergot or @gluck validate the PR when comfortable.

cheers